test.js

```diff
@@ -172,3 +172,44 @@ test.serial('if no options are assigned, behave as silent', t => {
 	returnValue = true;
 	t.deepEqual(log, []);
 });
+
+test.serial('if once option is true, only the first write is silent', t => {
+	let returnValue;
+	const log = [];
+
+	process.stdout = {
+		write: loggingWrite(log, () => returnValue)
+	};
+
+	m.stdout({once: true}, str => str);
+
+	process.stdout.write('foo');
+	process.stdout.write('bar');
+	process.stdout.write('unicorn');
+
+	t.deepEqual(log, [['bar'], ['unicorn']]);
+});
+
+test.serial('if once option is true and silent is false, hook prints the first write and std the next write ', t => {
+	let hookReturnValue;
+	const log = [];
+
+	process.stdout = {
+		write: loggingWrite(log, () => true)
+	};
+
+	m.stdout({silence: false, once: true}, str => {
```

Comment:
Done. After fixing the option name, the test was failing because it was slightly wrong. Had to fix the test.